apps/i18n/common/en_us.json

```diff
@@ -614,6 +616,9 @@
   "featureUnpublishedWarning": "This project is currently unpublished. It can still be marked as featured, but it will not show in the gallery until the owner publishes it.",
   "feedback": "Feedback",
   "feedbackCommentAreaHeader": "Teacher Feedback",
+  "feedbackDownloadFileName": "Feedback for {sectionName} in {scriptName} on {date}.csv",
+  "feedbackDownloadOverview": "This CSV file contains all feedback you’ve completed for your section {sectionName} in levels within **{scriptName}**. You can leave feedback your students by going to a level in this unit, viewing a students work, and clicking the \"Feedback\" tab.",
```

Comment:
Two little typos:  
"You can leave feedback your students" -> "You can leave feedback for your students"
"viewing a students work" -> "viewing a student's work"